composer.json

```diff
@@ -12,15 +12,18 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "aws/aws-sdk-php": "~2.4",
-        "illuminate/support": "~5.0"
+        "php": ">=5.5.9",
+        "aws/aws-sdk-php": "~3.0",
+        "illuminate/support": "~5.1"
```

Comment:
Laravel is not likely to change any parts that are relevant for a while. SDKv3 is the reason we had to bump to 5., so I think ~5.1 will be fine. If it ever becomes not fine, then we'll worry about it at that time.